certbot/certbot/_internal/snap_config.py

```diff
@@ -0,0 +1,87 @@
+"""Module configuring Certbot in a snap environment"""
+import socket
+import sys
+
+from requests import Session
+from requests.adapters import HTTPAdapter
+from urllib3.connection import HTTPConnection
+from urllib3.connectionpool import HTTPConnectionPool
+
+from acme.magic_typing import List
+
+from certbot.compat import os
+
+
+def prepare_env(cli_args):
+    # type: (List[str]) -> List[str]
+    snap_arch = os.environ.get('SNAP_ARCH')
+    if snap_arch == 'arm64':
```

Comment:
How about using a dictionary for this instead?

Reply:
Totally! I was still designing this from the bash script...